### PR TITLE
Feature/r3.1 update license dependencies

### PR DESCRIPTION
**Pull request by awholegunch**
Updated license dependency source, output and generation tool files.
Added BSD licenses to license page that require publication as condition of license.

Fix for https://issues.cask.co/browse/CDAP-3082

Building...will be available when finished at:

- [Licenses](http://docs-staging.cask.co/cdap/3.1.0-SNAPSHOT-feature-r3.1_Update_License_Dependencies/en/reference-manual/licenses/index.html)

### CODE REVIEW

File: cdap-docs/reference-manual/source/licenses/cdap-enterprise-dependencies.rst [hunk 1]
```diff
@@ -24,22 +24,12 @@ Cask Data Application Platform Distributed Dependencies
    "antlr.antlr-2.7.7.jar","2.7.7","\-","ANTLR 3 License (BSD License)","http://www.antlr2.org/license.html"
    "aopalliance.aopalliance-1.0.jar","1.0","\-","Public Domain","http://aopalliance.sourceforge.net/"
    "c3p0.c3p0-0.9.1.1.jar","0.9.1.1","\-","EPL, LGPLv2",""
-   "cdap-etl-batch-3.0.0-SNAPSHOT.jar","3.0.0","SNAPSHOT","Apache License, Version 2.0","http://www.apache.org/licenses/LICENSE-2.0.html"
-   "cdap-etl-realtime-3.0.0-SNAPSHOT.jar","3.0.0","SNAPSHOT","Apache License, Version 2.0","http://www.apache.org/licenses/LICENSE-2.0.html"
+   "cdap-etl-batch-3.2.0-SNAPSHOT.jar","3.2.0","SNAPSHOT","Apache License, Version 2.0","http://www.apache.org/licenses/LICENSE-2.0.html"
```
Comment (chtyim) [inline cdap-docs/reference-manual/source/licenses/cdap-enterprise-dependencies.rst]: 3.2.0-SNAPSHOT?

Comment (awholegunch) [inline cdap-docs/reference-manual/source/licenses/cdap-enterprise-dependencies.rst]: I know. I have asked Sree about this...

**Comment by awholegunch** [comment]
On-hold because we need to resolve the issue of the 3.2 dependencies that showed up on the integration cluster.

File: cdap-docs/reference-manual/source/licenses/cdap-enterprise-dependencies.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/licenses/cdap-enterprise-dependencies.rst]: Opened issue https://issues.cask.co/browse/CDAP-3135 as blocker.

**Comment by awholegunch** [comment]
Problem was the cluster I used was not a 3.1 cluster. Closing the PR and redoing the files.

**Pull request closed without merging**